lib/src/date-input/types.ts

```diff
@@ -57,14 +57,14 @@ type Props = {
    * If the string value is a valid date, error will
    * be null. Also, if the string value is not a valid date, date will be null.
```

Comment:
The comment needs to be updated as the documentation

Reply:
Done!